Lib/fontTools/ttLib/tables/otConverters.py

```diff
@@ -332,6 +334,21 @@ def xmlWrite(self, xmlWriter, font, value, name, attrs):
 		xmlWriter.newline()
 
 
+class STATFlags(UShort):
```

Comment:
Maybe a generic NamedFlags converter that could be defined in otConverters and used anywhere there are some bit flags that we want to be accompanied by a comment with the flag names. Perhaps add a TODO for later.

Reply:
Updated to do the same as lookup flags.